Locale parsing seems broken on Linux for non-en locales

Trying to compile for the it_IT locale on Linux gives me this

``````
LC_NUMERIC=it_IT locale -ck LC_NUMERIC | node src/locale.js src/core/format-locale.js > src/core/format-it_IT.js
./node_modules/uglify-js/bin/uglifyjs d3.core.js.tmp -b indent-level=2 -o d3.core.js
WARN: ERROR: Unexpected token: string () [d3.core.js.tmp:2,37]

/home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:199
    throw new JS_Parse_Error(message, line, col, pos);
          ^
Error
    at new JS_Parse_Error (/home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:185:18)
    at js_error (/home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:199:11)
    at croak (/home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:630:9)
    at token_error (/home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:638:9)
    at unexpected (/home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:644:9)
    at semicolon (/home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:664:43)
    at /home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:788:38
    at /home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:677:24
    at /home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:1392:23
    at Object.parse (/home/ludo/tmp/d3/node_modules/uglify-js/lib/parse.js:1402:7)
make: *** [d3.core.js] Error 1
rm src/core/format-it_IT.js```
``````

`LC_NUMERIC=it_IT locale -ck LC_NUMERIC` gives

```
LC_NUMERIC
decimal_point=","
thousands_sep=""
grouping=-1;-1
numeric-decimal-point-wc=44
numeric-thousands-sep-wc=0
numeric-codeset="UTF-8"
```

--- a/src/locale.js
+++ b/src/locale.js
@@ -3,7 +3,7 @@ var fs = require("fs"),
     formats = {},
     kvRe = /=/,
     valueRe = /;/g,
-    quotedRe = /"([^"]+?)"/g,
+    quotedRe = /"([^"]*?)"/g,
     data = [];
 
 process.stdin.resume();

